themes/ignite/layouts/partials/SideBar.html

```diff
@@ -31,7 +31,7 @@
   <div class="border-b lg:hidden dark:border-dark-200">{{ partial "NavBar.html" . }}</div>
   <nav class="py-3 px-1">
     {{ if eq .Section "blog" }}
-      <h5 class="font-semibold py-1 uppercase">{{ .Section }}</h5>
+      <h5 class="font-semibold py-1 uppercase">{{ .Section | title }}</h5>
```

Comment:
what this does ?

Reply:
It changes `blog` to `Blog` which will be used in docsearch level 0